### PR TITLE
Update screen id name for generating new and random address

### PR DESCRIPTION
**Pull request by shekhar-cis**
Update id of random screen; which generate the random identity.

### CODE REVIEW

**Comment by PeterSurda** [comment]
This PR is ok but too small. I'd prefer multiple similar changes in one PR.

**Comment by shekhar-cis** [comment]
> This PR is ok but too small. I'd prefer multiple similar changes in one PR.

ok

File: src/bitmessagekivy/baseclass/maildetail.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def assign_mail_details(self, data):
             self.status = data[0][4]
         self.time_tag = ShowTimeHistoy(data[0][4]) if self.kivy_state.detailPageType == 'inbox' \
             else ShowTimeHistoy(data[0][6])
-        self.avatarImg = os.path.join(self.kivy_state.imageDir, 'avatar.png') \
```
Comment (PeterSurda) [inline src/bitmessagekivy/baseclass/maildetail.py:138]: just don't use "avatar", this isn't an avatar.

**Review by PeterSurda — CHANGES_REQUESTED** [review]
normally, the "draft" change would be one PR (or at least one commit), and the id/sc changes would be another one, but for all files together.

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**